Makefile

```diff
@@ -48,7 +48,7 @@ RELEASE_CPP_LIB_OBJECTS := $(subst build/release-cpp/main.o,,$(RELEASE_CPP_OBJEC
 all: release
 
 clean:
-	@rm -rf build wren wrend libwren libwrend
+	@rm -rf build wren wrend libwren* libwrend*
```

Comment:
I agree on the heebie-jeebie aspect of it; seems like a code smell. I'll make it more explicit.